cms/djangoapps/contentstore/views/course.py

```diff
@@ -1312,6 +1313,11 @@ def advanced_settings_handler(request, course_key_string):
     course_key = CourseKey.from_string(course_key_string)
     with modulestore().bulk_operations(course_key):
         course_module = get_course_and_check_access(course_key, request.user)
+
+        advanced_dict = CourseMetadata.fetch(course_module)
+        if IgnoreMobileAvailableFlagConfig.is_enabled() and 'edx.org' in request.get_host():
```

Comment:
I don't think we need `edx.org` check here. It should be hidden if flag is enabled.